ecs-cli/modules/cli/local/ps_app.go

```diff
@@ -63,18 +68,39 @@ const (
 // If the --all flag is provided, then list all local ECS task containers.
 // If the --json flag is provided, then output the format as JSON instead.
 func Ps(c *cli.Context) {
+	if err := optionsPreCheck(c); err != nil {
+		logrus.Fatalf("Tasks can be either created by local files or remote files")
+	}
 	containers := listContainers(c)
 	displayContainers(c, containers)
 }
 
+func optionsPreCheck(c *cli.Context) error {
```

Comment:
I'd recommend renaming this to `validateOptions` or `validateInput` or `validateFlags`

Reply:
ok I will use validateOptions